geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -308,6 +308,9 @@ object RichSimpleFeatureType {
     def getTableSplitterOptions: Map[String, String] =
       userData[String](TABLE_SPLITTER_OPTS).map(new KVPairParser().parse).getOrElse(Map.empty)
 
+    def setZShards(splits: Int) : Unit = sft.getUserData.put(NUM_SPLITS_KEY, splits.toString)
+    def getZShards: Int = Option[Int](Try(userData[String](NUM_SPLITS_KEY).get.toInt).getOrElse(4)).get
```

Comment:
could rewrite this as:
```
userData[String](NUM_SPLITS_KEY).map(_.toInt).getOrElse(4)
```
generally we don't bother with catching exceptions in the rest of these methods... maybe we should revisit that but in general these are advanced options that we don't expect to fail.